dms/tests/test_directory.py

```diff
@@ -36,7 +36,7 @@ def test_copy_root_directory(self):
             self.directory_root_demo_03.count_files, copy_root_directory.count_files
         )
 
-    @multi_users(lambda self: self.multi_users(demo=False), callback="_setup_test_data")
+    @multi_users(lambda self: self.multi_users(), callback="_setup_test_data")
```

Comment:
Why removing this then?

Reply:
Originally it was not here, and It was added because an error was raised, but the error was raised due to an error to the mimetype detection. Travis was detecting that one of the files of the folder was an SVG. Now we are using a folder that does not has this kind of records and is not giving problems.
So, it was a mistake to add `demo=False`. This PR solves the issue and now it is consistent with 13.0

Reply:
OK